### PR TITLE
Scheduler: add the 'All Day Panel Mode' demo description

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: JSDemos/Demos/Scheduler/AllDayPanelMode/description.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+The **All day** Scheduler panel displays appointments that last all day or have the [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay) property set to **true**. To change the **All day** panel display mode, assign one of the following values to the [allDayPanelMode](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#allDayPanelMode) property:
```
Comment (Abadzhev) [inline JSDemos/Demos/Scheduler/AllDayPanelMode/description.md]: Did you mean to say something like this in the first sentence? 

```suggestion
The **All day** Scheduler panel displays appointments that last over 24 hours or have the [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay) property set to **true** (don't have specific start/end times). To change the **All day** panel display mode, assign one of the following values to the [allDayPanelMode](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#allDayPanelMode) property:
```

Or maybe this: 

```suggestion
The **All day** Scheduler panel displays appointments that last all day (have the [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay) property set to **true**). To change the **All day** panel display mode, assign one of the following values to the [allDayPanelMode](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#allDayPanelMode) property:
```

File: JSDemos/Demos/Scheduler/AllDayPanelMode/description.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+- *'all'*    
+Displays all appointments that have the [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay) property set to **true** and the ones that have the length equal to or more than 24 hours. The Scheduler does not show these appointments on the view.
```
Comment (Abadzhev) [inline JSDemos/Demos/Scheduler/AllDayPanelMode/description.md]: Stronger verb allows to shorten the sentence: "have the length equal to or more than 24 hours" -> "last 24 hours or longer".

More often than not, Ray replaces "show" with "display" when we talk about our controls and their elements. Also "in the view" reads to me a bit better than "on the view". But maybe both are okay. 

Some of these comments apply to other list items.

```suggestion
Displays all appointments that have the [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay) property set to **true** and the ones that last 24 hours or longer. The Scheduler does not display these appointments in the view.
```

File: JSDemos/Demos/Scheduler/AllDayPanelMode/description.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+The **All day** Scheduler panel displays appointments that last over 24 hours or have the [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay) property set to **true** (don't have specific start/end times). To change the **All day** panel display mode, assign one of the following values to the [allDayPanelMode](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#allDayPanelMode) property:
+
+- *'all'*    
+Displays all appointments that have the [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay) property set to **true** and the ones that last 24 hours or longer. The Scheduler does not display these appointments in the view.
+
+- *'allDay'* (default)   
+Displays only appointments that have the [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay) property set to **true**. The Scheduler shows other appointments on the view.
```
Comment (vladaskorohodova) [inline JSDemos/Demos/Scheduler/AllDayPanelMode/description.md]:
```suggestion
Displays only appointments that have the [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay) property set to **true**. The Scheduler shows other appointments in the view.
```

File: JSDemos/Demos/Scheduler/AllDayPanelMode/description.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+The **All day** Scheduler panel displays appointments that last over 24 hours or have the [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay) property set to **true** (don't have specific start/end times). To change the **All day** panel display mode, assign one of the following values to the [allDayPanelMode](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#allDayPanelMode) property:
+
+- *'all'*    
+Displays all appointments that have the [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay) property set to **true** and the ones that last 24 hours or longer. The Scheduler does not display these appointments in the view.
+
+- *'allDay'* (default)   
+Displays only appointments that have the [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay) property set to **true**. The Scheduler shows other appointments in the view.
+
+- *'hidden'*    
+Hides the **All day** panel. The Scheduler shows all appointments on the view.
```
Comment (vladaskorohodova) [inline JSDemos/Demos/Scheduler/AllDayPanelMode/description.md]:
```suggestion
Hides the **All day** panel. The Scheduler shows all appointments in the view.
```

File: JSDemos/Demos/Scheduler/AllDayPanelMode/description.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+The **All day** Scheduler panel displays appointments that last over 24 hours or have the [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay) property set to **true** (don't have specific start/end times). To change the **All day** panel display mode, assign one of the following values to the [allDayPanelMode](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#allDayPanelMode) property:
```
Comment (dirk-pieterse) [inline JSDemos/Demos/Scheduler/AllDayPanelMode/description.md]:
```suggestion
The **All day** Scheduler panel displays appointments that last longer than 24 hours or have the [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay) property set to **true** (don't have specific start/end times). To change the **All day** panel display mode, assign one of the following values to the [allDayPanelMode](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#allDayPanelMode) property:
```

**Review by dirk-pieterse — DISMISSED** [review]

**Review by AnnaKondratova — APPROVED** [review]

**Pull request merged**